speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/racescreens/raceresultsmenus.cpp

```diff
@@ -712,18 +722,18 @@ RmShowResults(void *prevHdle, tRmInfo *info)
 			}
 			
 			if (bQualif)
```

Comment:
add above the switch statement: // SIMULATED DRIVING ASSISTANCE: added RaceEndType parameters